lib/middlewares/auth.ts

```diff
@@ -61,7 +61,10 @@ class Auth extends AbstractMiddleware {
         const sdk = await import('@voiceflow/sdk-auth/express').catch(() => null);
         if (!sdk) return next();
 
-        return sdk?.createAuthGuard(client)(actions as any[])(req, res, next);
+        // we cant return immediatly here or the catch is not caught properly
+        // eslint-disable-next-line sonarjs/prefer-immediate-return
+        const authorizedNext = await sdk?.createAuthGuard(client)(actions as any[])(req, res, next);
+        return authorizedNext;
```

Comment:
```suggestion
        return await sdk?.createAuthGuard(client)(actions as any[])(req, res, next);
```

We could just an an `await` here too.

Reply:
Maybe it's worth updating the SDK to call `next` if it catches an error so we don't have to worry about it during implementation.

Reply:
yes! maybe something simple we can add to a utility sprint? there was a list of these smaller improvements to do, but cant find it